YoYoProject/Controllers/GMSprite.cs

```diff
@@ -379,7 +382,21 @@ public GMSpriteFrame Get(Guid id)
             public void Delete(GMSpriteFrame frame)
             {
                 frames.Remove(frame);
+                string framepath = Path.Combine(frame.Project.RootDirectory, "sprites", frame.Sprite.Name, frame.Id.ToString(), ".png");
+                try
```

Comment:
Fair, but you should make this a static method on FileSystem in YoYoProject.Utility.
FileSystem.SafeDelete() or something.

Reasoning being that it's additional clutter in the code that obscures the intent.